src/RestApi/StoreApi/docs/cart.md

```diff
@@ -0,0 +1,444 @@
+          "sizes": "(max-width: 800px) 100vw, 800px",
+          "name": "pennant-1.jpg",
+          "alt": ""
+        }
+      ],
+      "variation": [],
+      "prices": {
+        "currency_code": "GBP",
+        "currency_symbol": "£",
+        "currency_minor_unit": 2,
+        "currency_decimal_separator": ".",
+        "currency_thousand_separator": ",",
+        "currency_prefix": "£",
+        "currency_suffix": "",
+        "price": "1105",
+        "regular_price": "1105",
+        "sale_price": "1105",
+        "price_range": null,
+        "raw_prices": {
+          "precision": 6,
+          "price": "11050000",
+          "regular_price": "11050000",
+          "sale_price": "11050000"
+        }
+      },
+      "totals": {
+        "currency_code": "GBP",
+        "currency_symbol": "£",
+        "currency_minor_unit": 2,
+        "currency_decimal_separator": ".",
+        "currency_thousand_separator": ",",
+        "currency_prefix": "£",
+        "currency_suffix": "",
+        "line_subtotal": "1105",
+        "line_subtotal_tax": "0",
+        "line_total": "884",
+        "line_total_tax": "0"
+      }
+    }
+  ],
+  "items_count": 2,
+  "items_weight": 0,
+  "needs_payment": true,
+  "needs_shipping": true,
+  "totals": {
+    "currency_code": "GBP",
+    "currency_symbol": "£",
+    "currency_minor_unit": 2,
+    "currency_decimal_separator": ".",
+    "currency_thousand_separator": ",",
+    "currency_prefix": "£",
+    "currency_suffix": "",
+    "total_items": "2105",
+    "total_items_tax": "0",
+    "total_fees": "0",
+    "total_fees_tax": "0",
+    "total_discount": "421",
+    "total_discount_tax": "0",
+    "total_shipping": "500",
+    "total_shipping_tax": "0",
+    "total_price": "2184",
+    "total_tax": "0",
+    "tax_lines": []
+  },
+  "errors": []
+}
+```
+
+### Error Response
+
+If a cart action cannot be performed, an error response will be returned. This will include a reason code and an error message:
+
+```json
+{
+  "code": "woocommerce_rest_cart_invalid_product",
+  "message": "This product cannot be added to the cart.",
+  "data": {
+    "status": 403
+  }
+}
+```
+
+Some error responses indicate conflicts (error 409), for example, when an item cannot be found or a coupon is no longer applied. When this type of response is returned, the current state of the cart from the server is also returned as part of the error data:
+
+```json
+{
+  "code": "woocommerce_rest_cart_invalid_key",
+  "message": "Cart item no longer exists or is invalid.",
+  "data": {
+    "status": 409,
+    "cart": { ... }
+  }
+}
+```
+
+This allows the client to remain in sync with the cart data without additional requests.
```

Comment:
Hmm, but a request is needed to get this data right? I'm guessing you mean "without additional full WordPress route load" here?

Reply:
> Hmm, but a request is needed to get this data right

an additional request would be needed if you still want to do the operation, it means if your cart is outdated, it will get updated.